### PR TITLE
vibe and chat

### PR DESCRIPTION
**Pull request by Kanan11**

### CODE REVIEW

File: src/frontend/components/PubPage.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          {peers.map(friend => (
+            <div className='friendsListWrap2' key={friend.pk}>
+              <div className='column friendsList-2'>
+                <div className='column friendsIcon'>
+                  <span className='icon-3'>{friend.picture}</span>
+                </div>
+                <div className='column'>
+                  <h1>{friend.name}</h1>
+                  <span className='icon-2'>{icons[friend.sex]}</span>
+                  <h3 className='smalleTagline'>{friend.tagline}</h3>
```
Comment (telamon) [inline src/frontend/components/PubPage.jsx:38]: don't use `h3` for small text. `h3` means "Header3" which makes text larger than normal.
instead use:
```jsx
<div><small>{friend.tagline}</small></div>
```
if you still need a css-class then fix case and typo: `small-tagline`

File: src/frontend/components/PubPage.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      })
+  }
+  return (
+    <>
+      <div className='hero'>
+        <h1 className='bar'>Pub name is <strong>"HardRock"</strong></h1>
+        <h3 className='membersList'>Here is memmbers list which is at this bar now</h3>
+        <VibeList />
+        <div className='friendsList friendsListWrap'>
+          {peers.map(friend => (
```
Comment (telamon) [inline src/frontend/components/PubPage.jsx:29]: refactor for consistency:
rename `friend` => `peer`

`useFriendsList()` will be renamed in future to `usePeers()`

File: src/frontend/components/Pubs.jsx [hunk 3]
```diff
@@ -1,15 +1,42 @@
...
   const history = useHistory()
+  console.log(pubsList)
+  function Pub (pub) {
+    const icons = {
+      beer: '🍺',
+      wine: '🍷',
+      cider: '🍸'
+    }
+    function click () {
+      console.log('pub name clicked', pub.name)
```
Comment (telamon) [inline src/frontend/components/Pubs.jsx:16]: remove line 16

File: src/frontend/components/Pubs.jsx [hunk 4]
```diff
@@ -1,15 +1,42 @@
...
+      enterPub(pub.name).then(() => {
+        console.log('pub entered', pub.name)
+        history.push('/pub')
+      }).catch(err => {
+        console.error('faild entered Pub', err)
```
Comment (telamon) [inline src/frontend/components/Pubs.jsx:21]: Fix typo `faild entered Pub` => `Failed entering pub`

File: src/frontend/components/RulesPage.jsx [hunk 5]
```diff
@@ -7,7 +7,7 @@ const Rules = () => {
     <>
       <h2>Policy of PicoChat</h2>
```
Comment (telamon) [inline src/frontend/components/RulesPage.jsx:8]: Replace `h2` and `h4` with:
```jsx
<h2>Disclaimer</h2>`
<p>At this point in time we have no control over any peer to peer interactions and
by design might never have.
<br />
We do not use cookies.
<br />
And we do not store any personal information.
PicoChat does not make use of a server nor a central-database.
All information that you input is in your own browser.
</p>
<p>
<sub>bottomline</sub>
<strong>We take no responsibility for how this software is used
or distributed.</strong>
<br />
<small><a href="https://github.com/telamon/picochat/issues">(any questions?)</a></small>
</p>

File: public/global.css [hunk 6]
```diff
@@ -50,3 +50,87 @@ body {
...
+}
+
+.friendsList-2{
+  border-style: solid;
+  margin: 2vw;
+  width: 25vw;
+  text-align: center;
+}
+
+.friendsList{
```
Comment (telamon) [inline public/global.css:116]: `.peers-list`

**Review by telamon — CHANGES_REQUESTED** [review]
Awesome!!!!! :beers:

**Review by telamon — APPROVED** [review]
Good job!

**Comment by Kanan11** [comment]
wip is done

**Pull request merged**